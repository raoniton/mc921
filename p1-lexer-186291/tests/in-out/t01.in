class Main {
    public static void ma$in() {
        return;
    }
}