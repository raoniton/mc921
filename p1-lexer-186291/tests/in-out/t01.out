LexToken(CLASS,'class',1,0)
LexToken(ID,'Main',1,6)
LexToken(LBRACE,'{',1,11)
LexToken(PUBLIC,'public',2,17)
LexToken(STATIC,'static',2,24)
LexToken(VOID,'void',2,31)
LexToken(ID,'ma',2,36)
Lexical error: Illegal character '$' at 2:26
LexToken(ID,'in',2,39)
LexToken(LPAREN,'(',2,41)
LexToken(RPAREN,')',2,42)
LexToken(LBRACE,'{',2,44)
LexToken(RETURN,'return',3,54)
LexToken(SEMI,';',3,60)
LexToken(RBRACE,'}',4,66)
LexToken(RBRACE,'}',5,68)
